modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenOperation.java

```diff
@@ -17,6 +17,7 @@
 
 package org.openapitools.codegen;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
```

Comment:
@jlengrand thanks for the PR. This line of import is no longer needed after renaming the function below, right?

Reply:
Very good point! It's a remnant of the old method